packages/generator-react-server/generators/app/templates/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-preset-react-server": "^0.3.3",
+    "babel-preset-react-server": "^0.4.10",
```

Comment:
Oh, yikes.  Totally forgot about this thing... I wonder if we can use Asini's configurable package locations to get this updating automatically. :thinking: